lib/fastlane/plugin/appcenter/actions/appcenter_fetch_version_number.rb

```diff
@@ -0,0 +1,99 @@
+require 'json'
+require 'net/http'
+require 'fastlane_core/ui/ui'
+
+module Fastlane
+  UI = FastlaneCore::UI unless Fastlane.const_defined?("UI")
+
+  module Actions
+    class AppcenterFetchVersionNumberAction < Action
+      def self.description
+        "Fetches the latest version number of an app from App Center"
+      end
+
+      def self.authors
+        ["jspargo", "ShopKeep"]
+      end
+
+      def self.run(params)
+        api_token = params[:api_token]
+        app_name = params[:app_name]
+        owner_name = params[:owner_name]
+
+        if owner_name.nil?
```

Comment:
Since the owner name is now required, this check and the method solving the missing value are redundant